### PR TITLE
Make compatible with relay v6.

### LINKED ISSUE #17: Cannot find module 'relay-compiler/lib/RelayCompilerMain'
I'm trying to set up this plugin and I'm getting such error in vs console:

```
[ERROR] A config file failed to load with options: {"configPath":"/Users/user/app","requireConfig":true}.
    The error was: Error: Cannot find module 'relay-compiler/lib/RelayCompilerMain'
    at Object.loadConfig (/Users/user/.vscode/extensions/apollographql.vscode-apollo-1.10.0/node_modules/apollo-language-server/lib/config/loadConfig.js:38:34)
	at client.onReady.then.client.onNotification.message (/Users/user/.vscode/extensions/apollographql.vscode-apollo-1.10.0/lib/extension.js:44:35)
	at handleNotification (/Users/user/.vscode/extensions/apollographql.vscode-apollo-1.10.0/node_modules/vscode-jsonrpc/lib/main.js:489:43)
	at processMessageQueue (/Users/user/.vscode/extensions/apollographql.vscode-apollo-1.10.0/node_modules/vscode-jsonrpc/lib/main.js:260:17)
	at Immediate.setImmediate [as _onImmediate] (/Users/user/.vscode/extensions/apollographql.vscode-apollo-1.10.0/node_modules/vscode-jsonrpc/lib/main.js:247:13)
```

My `apollo.config.js` is 

```ts
const { config } = require('vscode-apollo-relay').generateConfig();
module.exports = config;
```

Relay compiler version is `6.0.0`

Indeed, when I check content of `react-compiler` package: `node_modules/relay-compiler/lib` by running `cd` I get 

```
bin         core        index.js    reporters   util
codegen     handlers    language    transforms  validations
```

### PR DESCRIPTION
**Pull request by alloy**
Fixes #17

### CODE REVIEW

File: src/dependencies.ts [hunk 1]
```diff
@@ -51,3 +52,15 @@ try {
...
+
+let relayCompilerMainMod: typeof _RelayCompilerMain | null = null
+try {
+  // relay-compiler v6
+  relayCompilerMainMod = require("relay-compiler/bin/RelayCompilerMain")
+} catch (_) {
+  try {
+    // relay-compiler v5
+    relayCompilerMainMod = require("relay-compiler/lib/RelayCompilerMain")
+  } catch (_) {}
```
Comment (kastermester) [inline src/dependencies.ts]: Should we catch this error and state that we could not find a config at all?

**Comment by kastermester** [comment]
I ran this against our own codebase using `relay-compiler@5.x` - it worked as it should.

File: src/dependencies.ts (hunk 1 above)
Comment (alloy) [reply src/dependencies.ts]: Can do 👍

**Comment by alloy** [comment]
Thanks!

**Pull request merged**